CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This is the log of notable changes to EAS CLI and related packages.
 ### 🛠 Breaking changes
 
 ### 🎉 New features
+
 - Added support for SSO users. ([#1875](https://github.com/expo/eas-cli/pull/1875) by [@lzkb](https://github.com/lzkb))
```

Comment:
```suggestion
- Added support for SSO users. ([#1875](https://github.com/expo/eas-cli/pull/1875), [#1890](https://github.com/expo/eas-cli/pull/1890) by [@lzkb](https://github.com/lzkb))
```
If your previous changes for the same feature were not published yet, then just add a new PR to the existing changelog entry.

Reply:
They were already published, so cannot add to the existing entry

Reply:
Nothing was published, the last cli publish was 2 weeks ago.